tests/view_tests/tests/test_i18n.py

```diff
@@ -256,34 +257,68 @@ def test_i18n_with_locale_paths(self):
                     'este texto de app3 debe ser traducido')
 
 
-skip_selenium = not os.environ.get('DJANGO_SELENIUM_TESTS', False)
+class JavascriptI18nTestsMetaClass(type):
```

Comment:
Let's avoiding duplicating this metaclass and put it in a new file: `django/test/selenium.py`